model-archiver/model_archiver/tests/unit_tests/test_model_packaging.py

```diff
@@ -34,7 +34,8 @@ def __init__(self, **kwargs):
     export_path = '/Users/ghaipiyu/'
```

Comment:
I know this is not part of this PR, but this export path seems to be user specific. Can we clean it up and out into tmp folder so that it won't be in /Users folder? Or, alternatively, change it to current user's home (~/)?